### PR TITLE
64 - Create Panel core example

### LINKED ISSUE #64: Create Panel example that doesn't rely on edge-native-base

### PR DESCRIPTION
**Pull request by EdwinWiseOne**
Closes  #64 

Add an example for Panel that doesn't rely on the edge-native-base.

Tested in edge-product.

### CODE REVIEW

**Review by andrewcollette — APPROVED** [review]

File: Core/Panel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+# IMPORTANT: Please do not define any of the EDGE_* environment variables, or
+# any of the JUPYTERHUB_* variables, directly in this file. They will be set
+# automatically when running in Edge, or by the "ci" module when running
+# locally.
+
+# First stage
+
+ARG BASE_IMAGE=quay.io/enthought/edm-centos-7:3.4.0
```
Comment (andrewcollette) [inline Core/Panel/Dockerfile:20]: We could update the version number here, I think (we are past 3.4.0).

File: Core/Panel/README.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+## Getting EdgeSession to work in development
+
+When you run your app on Edge, you can create an EdgeSession object simply by
+calling the constructor (``mysession = EdgeSession()``).  This works by
+collecting environment variables set by Edge when the container is launched.
+
+When developing locally, it's also convenient to have an EdgeSession.  You
+can get the "ci" module to inject the appropriate environment variables, so
+that your ``EdgeSession()`` call will work with ``python -m ci run`` and
+``python -m ci preflight``.
```
Comment (andrewcollette) [inline Core/Panel/README.md:79]: Reference to `preflight` should be removed.

File: Core/Panel/README.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+To do so, follow this procedure:
+
+* Copy the "dev_settings.json.example" file to "dev_settings.json".
+* Define EDGE_API_SERVICE_URL in that file.  The typical value here is
+  ``"https://edge.enthought.com/services/api"``.
+* Define EDGE_API_ORG in that file.  This is the "short name" displayed in
+  the URL bar when you log into an organization, for example, ``"default"``.
+* Define EDGE_API_TOKEN.  You can get one of these by going to the
+  ``/hub/token`` endpoint on the Edge server.
```
Comment (andrewcollette) [inline Core/Panel/README.md:89]: This should be updated; we have an actual UI for this now.

File: Core/Panel/README.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+## Guidelines for your Dockerfile
+
+Edge will run your app next to a built-in reverse proxy, which allows
+you to skip a lot of work in the development process.  This includes stripping
+the prefix from requests, handling the OAuth2 login flow, pinging JupyterHub
+for container activity, and more.  But, there are a few guidelines you will
+need to follow in your own Dockerfile.
+
+* Don't change the user to anything other than ``app`` (for example, by the
```
Comment (andrewcollette) [inline Core/Panel/README.md:108]: I think this doesn't apply anymore, since it only existed to keep edge-native-base happy.

**Review by andrewcollette — APPROVED** [review]
Sorry, my comments got lost.  Just a few.

**Comment by EdwinWiseOne** [comment]
Once this is in I can also migrate the changes to the other examples.

File: Core/Panel/Dockerfile (hunk 1 above)
Comment (EdwinWiseOne) [reply Core/Panel/Dockerfile:20]: Everywhere in Edge we use :latest (except for src_proxy which copied the version from here, so I'll update that too).

**Pull request merged**